.circleci/config.yml

```diff
@@ -181,6 +291,56 @@ jobs:
           bump: prerelease
           environment: none
+          rebase_with_master: true
+
+  notify_about_merge_to_master:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - run:
+          name: "Trigger build on branches with PRs (because master had a commit)"
+          command: |
+            TARGET_USER="AdaptiveConsulting"
+            TARGET_BRANCH="master"
+
+            # Fetching info about opened pull requests (using github API)
+            # PLEASE NOTE: calling GitHub API with token restricts amount of calls to 60 per hour:
+            # https://developer.github.com/v3/#rate-limiting
+            pr_info=$(curl https://api.github.com/repos/AdaptiveConsulting/ReactiveTraderCloud/pulls\?state=open\?access_token\=${GIT_API_USER_TOKEN})
+
+            # Iterating through the list of opened PRs and mapping records to objects that only have info about PR number and full branch name
+            for pr_item in $(jq -c 'map( {pr_number: .number, branch_label: .head.label} ) | .[]' \<<< $pr_info); do
+
+              # Extracting PR number and PR label from JSON object into separate variables
+              pr_number=$(jq -c '.pr_number' \<<< $pr_item)
+              branch_label=$(jq -c '.branch_label' \<<< $pr_item)
+
+              # regex-ing $branch_label to see if it is located in original repo (not in a fork).
+              # For this we are testing if it starts with the main repo org name ("AdaptiveConsulting" in our case)
+              NON_FORKED_BRANCH_NAME=$( grep -oP "(?<=\"${TARGET_USER}:).*?(?=\")" \<<< $branch_label) || true
+              if [ ! -z "$NON_FORKED_BRANCH_NAME" ]
+              then
+                echo "Using non-forked CircleCi branch: ${NON_FORKED_BRANCH_NAME}"
+
+                # If it is non-forked branch, org name will be removed with Regex:
+                # "AdaptiveConsulting:some-branch" => "some-branch"
+                # ^ this is how CircleCi API wants us to trigger builds on non-forked branches (by branch name only)
+                CIRCLE_BRANCH_TO_TRIGGER_BUILD=$NON_FORKED_BRANCH_NAME
+              else
+                # if it's a forked branch, create a branch name based on the PR number.
+                # Please note that we are adding extra "head" suffix to work around existing bug in CircleCi API v2
+                # (see description in "Fetch git origin remote" command above)
+                FORKED_BRANCH_NAME="pull/"$pr_number"/head"
+                echo "Using forked CircleCi branch: ${FORKED_BRANCH_NAME}"
+                CIRCLE_BRANCH_TO_TRIGGER_BUILD=$FORKED_BRANCH_NAME
+              fi
+
+              echo "Triggering build: ${CIRCLE_BRANCH_TO_TRIGGER_BUILD}"
+
+              curl -u ${CIRCLE_API_USER_TOKEN}: -X POST --header "Content-Type: application/json" -d "{
+                \"branch\": \"${CIRCLE_BRANCH_TO_TRIGGER_BUILD}\"
+              }" https://circleci.com/api/v2/project/github/AdaptiveConsulting/ReactiveTraderCloud/pipeline
```

Comment:
I think

`https://circleci.com/api/v2/project/github/AdaptiveConsulting/ReactiveTraderCloud/pipeline`

can be 

`https://circleci.com/api/v2/project/github/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/pipeline`